extensions/wikia/AdMixExperiment/scripts/noVideo1.js

```diff
@@ -0,0 +1 @@
+console.log('test ad mix');
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)